### ISSUE TITLE
ManaPlus___ManaPlus yb analysis

### ISSUE THREAD

**Issue by keheliya** [project-kotinos/issues#2]
Add a yb configuration file to https://github.com/project-kotinos/ManaPlus___ManaPlus
Instuctions are here: https://github.com/project-kotinos/root/blob/master/README.md 
If you come across problems, add them as comments to this issue.

**Comment by nictuku** [comment]
I got this

**Comment by nictuku** [comment]
A lot of the time on this project is spent building stuff inside docker and we don't easily support accelerating things there.
Example:
https://semaphoreci.com/4144/manaplus/branches/master/builds/438

**Comment by nictuku** [comment]
Oh, not in all CIs. https://github.com/project-kotinos/ManaPlus___ManaPlus/blob/master/.gitlab-ci.yml

**Comment by keheliya** [comment]
Hi @nictuku Have you pushed the yb YAML file to GitHub? I cannot see it in the repo.

**Comment by nictuku** [comment]
This was not meant to be closed. Still barely starting.

**Comment by nictuku** [comment]
@keheliya do you have details about which specific builds are we mirroring? They use different branches for different things, so the CI config may be in a different branch (the gitlab CI one is in a specific branch, for example).

Can I just pick the latest master builds?

**Comment by keheliya** [comment]
Yes. Pick the latest master branch build. I'll add this point to the instructions.

**Comment by nictuku** [comment]
This project's build is suuuuper complex. And they build a lot of things inside containers. I'll leave it for later.

**Comment by nictuku** [comment]
LOL GL HF John KTHNX
